src/models/users.js

```diff
@@ -0,0 +1,23 @@
+const userQuery = require('../../queries/users');
+
+const getAllUsers = () => {
+  const user = userQuery.getAllUsers();
+  return user.then(result => result.length < 1 ? { error: 'error retrieving users', status: 500 } : result)
+};
+
+const getUserById = (userId) => {
+  const user = userQuery.getUserById(userId);
+  return user.then(result => !result ? { error: "error retrieving user by id", status: 500 } : result)
```

Comment:
Doesn't this mean there was no user with that Id?

Reply:
issue corrected, sending again for approval